packages/mesos/extra/dcos-mesos-slave.service

```diff
@@ -23,4 +23,4 @@ ExecStartPre=/bin/ping -c1 leader.mesos
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-mesos-slave
 ExecStartPre=/opt/mesosphere/bin/make_disk_resources.py /var/lib/dcos/mesos-resources
 ExecStartPre=/bin/bash -c 'for i in /proc/sys/net/ipv4/conf/*/rp_filter; do echo 2 > $i; echo -n "$i: "; cat $i; done'
-ExecStart=$PKG_PATH/bin/mesos-agent
+ExecStart=$PKG_PATH/bin/start_mesos.sh $PKG_PATH/bin/mesos-agent
```

Comment:
There are couple `ExecStartPre` scripts here which seem to be setting some configuration for the agent (e.g., make_disk_resources.py). Should the fault domain configuration should also be done via a `ExecStartPre` script instead of `ExecStart`?  cc @klueska

Reply:
`ExecStartPre` executes before `ExecStart` and `ExecStart` is used to start daemon service

Reply:
@vinodkone sorry for confusion, i did not understand what you meant.
Ok, so currently there is no a good way to run a script from systemd unit and set the output if this script as environment variable.

`ExecStartPre=/opt/mesosphere/bin/make_disk_resources.py /var/lib/dcos/mesos-resources` this like does not set any environment variables. It might read [MESOS_RESOURCES](https://github.com/dcos/dcos/blob/master/packages/mesos/extra/make_disk_resources.py#L152), but it does not set it.